Example/NeatTipView/ButtonExtension.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  ButtonExtension.swift
+//  NeatTipView_Example
+//
+//  Created by sol manini on 10/1/19.
+//  Copyright © 2019 CocoaPods. All rights reserved.
+//
+
+import UIKit
+
+extension UIButton {
+  func configure(
+    bkgColor: UIColor = .aquamarineGreen,
+    borderColor: UIColor = .aquamarineGreen,
+    cornerRadius: CGFloat = 5,
+    borderWidth: CGFloat = 1,
+    horizontalPadding: CGFloat = 15,
+    verticalPadding: CGFloat = 10,
+    titleColor: UIColor = .white
+  ) {
+    backgroundColor = bkgColor
+    layer.cornerRadius = cornerRadius
+    layer.borderColor = borderColor.cgColor
+    layer.borderWidth = borderWidth
+    contentEdgeInsets = UIEdgeInsets(
+      top: verticalPadding,
+      left: horizontalPadding,
+      bottom: verticalPadding,
+      right: horizontalPadding
+    )
+    setTitleColor(titleColor, for: .normal)
+  }
+  
+  func addShadow(
```

Comment:
this should belong to a `UIView` extension

Reply:
Also aren't you missing `clipToBounds` or `maskToBounds` ?

Reply:
it is set to false by default, if I set the value to true is would limit the shadow to the layers bounds and therefore no shadow appears, right?

Reply:
Exactly, you should set it to false here, so if anyone uses this method in a view with the property in true it will work as expected.

Reply:
perfect!